pkg/features/openshift_features.go

```diff
@@ -5,11 +5,16 @@ import (
 )
 
 var RouteExternalCertificate featuregate.Feature = "RouteExternalCertificate"
+var MinimumKubeletVersion featuregate.Feature = "MinimumKubeletVersion"
```

Comment:
Could we rely solely on the operator to read the required feature and turn the authorizer on/off?

Reply:
that's what I was thinking we'd do, though I added this at @benluddy 's behest

Reply:
I left the feature gates in for now but would be happy to drop

Reply:
Please don't drop it. If we have to advise someone to disable the feature gate due to a bug, it would be a shame to then discover a second bug in the "unconfigured" mode of the authorizer or admission plugin without any operational mitigation.

Reply:
my understanding is CKASO sets all the feature gates it knows about https://github.com/openshift/cluster-kube-apiserver-operator/blob/8ef2c1d874971a0badd9d55b122d907f34dedc0d/pkg/cmd/render/render.go#L533 . I don't have a cluster up now, but I've frequently seen a log message in apiserver saying `config.go:124] Ignoring unknown FeatureGate "MinimumKubeletVersion"` because it wasn't taught about it. Once I get a cluster up, I'll verify the apiserver now recognizes the feature

Reply:
ah, okay, please verify it, thanks!

Reply:
as I thought:
```
I1127 17:45:46.642006      12 flags.go:64] FLAG: --feature-gates=":AWSClusterHostedDNS=true,:AWSEFSDriverVolumeMetrics=true,:AdditionalRoutingCapabilities=true,:AdminNetworkPolicy=true,:AlibabaPlatform=true,:AutomatedEtcdBackup=true,:AzureWorkloadIdentity=true,:BareMetalLoadBalancer=true,:BootcNodeManagement=true,:BuildCSIVolumes=true,:CSIDriverSharedResource=true,:ChunkSizeMiB=true,:CloudDualStackNodeIPs=true,:ClusterMonitoringConfig=true,:ConsolePluginContentSecurityPolicy=true,:DNSNameResolver=true,:DisableKubeletCloudCredentialProviders=true,:DynamicResourceAllocation=true,:EtcdBackendQuota=true,:Example=true,:ExternalOIDC=true,:GCPClusterHostedDNS=true,:GCPLabelsTags=true,:HardwareSpeed=true,:ImageStreamImportMode=true,:IngressControllerDynamicConfigurationManager=true,:IngressControllerLBSubnetsAWS=true,:InsightsConfig=true,:InsightsConfigAPI=true,:InsightsOnDemandDataGather=true,:InsightsRuntimeExtractor=true,:KMSv1=true,:MachineAPIProviderOpenStack=true,:MachineConfigNodes=true,:ManagedBootImages=true,:ManagedBootImagesAWS=true,:MaxUnavailableStatefulSet=true,:MetricsCollectionProfiles=true,:MinimumKubeletVersion=true,:MixedCPUsAllocation=true,:MultiArchInstallAWS=true,:MultiArchInstallGCP=true,:NetworkDiagnosticsConfig=true,:NetworkLiveMigration=true,:NetworkSegmentation=true,:NewOLM=true,:NodeDisruptionPolicy=true,:NodeSwap=true,:NutanixMultiSubnets=true,:OVNObservability=true,:OnClusterBuild=true,:OpenShiftPodSecurityAdmission=true,:PersistentIPsForVirtualization=true,:PinnedImages=true,:PlatformOperators=true,:PrivateHostedZoneAWS=true,:ProcMountType=true,:RouteAdvertisements=true,:RouteExternalCertificate=true,:ServiceAccountTokenNodeBinding=true,:SetEIPForNLBIngressController=true,:SignatureStores=true,:SigstoreImageVerification=true,:TranslateStreamCloseWebsocketRequests=true,:UpgradeStatus=true,:UserNamespacesPodSecurityStandards=true,:UserNamespacesSupport=true,:VSphereControlPlaneMachineSet=true,:VSphereDriverConfiguration=true,:VSphereMultiNetworks=true,:VSphereMultiVCenters=true,:VSphereStaticIPs=true,:ValidatingAdmissionPolicy=true,:VolumeAttributesClass=true,:VolumeGroupSnapshot=true,:ClusterAPIInstall=false,:ClusterAPIInstallIBMCloud=false,:EventedPLEG=false,:GatewayAPI=false,:MachineAPIMigration=false,:MachineAPIOperatorDisableMachineHealthCheckController=false,:MultiArchInstallAzure=false"

```
in the KAS args